pvlib/tests/test_conftest.py

```diff
@@ -19,3 +20,25 @@ def test_fail_on_pvlib_version_pass():
 @fail_on_pvlib_version('100000.0')
 def test_fail_on_pvlib_version_fail_in_test():
     raise Exception
+
+
+# set up to test passing arguments to conftest.fail_on_pvlib_version
```

Comment:
another minor nit: this comment could be improved, arguments are passed to the test, not the decorator